src/app/static/src/tests/root/getters.test.ts

```diff
@@ -349,7 +349,8 @@ describe("root getters", () => {
             modelOptions: mockModelOptionsState({warnings: undefined}),
             modelRun: mockModelRunState({warnings: undefined}),
             modelCalibrate: mockModelCalibrateState({warnings: undefined}),
-            genericChart: mockGenericChartState({warnings: undefined})
+            genericChart: mockGenericChartState({warnings: undefined}),
+            surveyAndProgram: mockSurveyAndProgramState({warnings: undefined})
```

Comment:
There should also be a test in this file that it gets all Review Inputs warnings when there are any?